widget/class-widget.php

```diff
@@ -0,0 +1,229 @@
+<?php
+
+// Block direct requests
+if ( !defined('ABSPATH') )
+   die('-1');
+
+/**
+ * Volunteer Management Widget Class
+ */
+class WI_Volunteer_Management_Widget extends WP_Widget {
+
+   /**
+   * Set up widget information
+   */
+   function __construct() {
+
+      // WP_Widget constructor accepts Base ID (string), Name (string), Widget Options (array), Control Options (array)
+      parent::__construct(
+         'WI_Volunteer_Management_Widget', // Base ID
+         __('Volunteer Management Widget', 'text_domain'), // Name
```

Comment:
Let's call this "Volunteer Management Opportunities". I figure that's a little more clear, and is consistent with other widgets by removing the word "Widget". 

We also want to make sure that the text domain is always set to "'wired-impact-volunteer-management'" instead of "text_domain".